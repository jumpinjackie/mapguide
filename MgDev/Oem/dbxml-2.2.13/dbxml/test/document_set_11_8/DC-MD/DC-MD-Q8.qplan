<XQuery>
  <Navigation>
    <QueryPlanFunction result="collection" container="11.8.1.dbxml">
      <OQPlan>n(P(edge-element-presence-none,=,root:http://www.sleepycat.com/2002/dbxml.order),P(node-element-presence-none,=,item_id),V(edge-attribute-equality-string,order.@id,=,'4'))</OQPlan>
    </QueryPlanFunction>
    <Step axis="child" name="order" nodeType="element"/>
    <DbXmlFilter>
      <Navigation>
        <Step axis="attribute" name="id" nodeType="attribute"/>
        <DbXmlCompare name="equal">
          <Sequence>
            <AnyAtomicTypeConstructor value="4" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
          </Sequence>
        </DbXmlCompare>
      </Navigation>
    </DbXmlFilter>
    <Step axis="descendant" name="item_id" nodeType="element"/>
  </Navigation>
</XQuery>
