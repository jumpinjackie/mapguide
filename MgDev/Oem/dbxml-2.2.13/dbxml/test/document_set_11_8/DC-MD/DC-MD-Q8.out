<item_id>601</item_id><item_id>580</item_id><item_id>899</item_id><item_id>5</item_id><item_id>223</item_id>